addons/website_sale/data/data.xml

```diff
@@ -121,6 +121,13 @@ response = DynamicFilter._get_products('alternative_products', model.env.context
             </field>
         </record>
         <!-- Dynamic Filter -->
+        <record id="dynamic_filter_cross_selling_alternative_products" model="website.snippet.filter">
+            <field name="action_server_id" ref="website_sale.dynamic_snippet_alternative_products"/>
+            <field name="field_names">display_name,description_sale,image_512</field>
+            <field name="limit" eval="16"/>
+            <field name="name">Alternative Products</field>
+            <field name="product_cross_selling">True</field>
+        </record>
```

Comment:
Data are in `noupdate`, so this will only work for new DB, but won't fix it for all the other.  =]

Reply:
@Demesmaeker 
Good Catch,  I have removed these changes. Now,  I have override  the `_fetchDynamicFilters` method of s_dynamic_snippet and added the last record. Currently the last record is `Alternative products`. Please check second commit.thank you